### ISSUE TITLE
GIBCT Estimate Your Benefits - VSP Accessibility Feedback - Design Intent

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7728]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Must

* Focus must be set on the "Your estimated benefits" after users press the "Calculate benefits" button. This ensures screen reader users know we have processed their form async and the estimate is ready to be reviewed.

## Should

* Both versions (VETTEC, mutiple filtering buckets) should have the text "Use the fields below to calculate your benefits". These are good instructions and all users will benefit from them.
* The "Your estimated benefits" header should not get the yellow focus halo when it receives focus. It's a non-native element, and this point is vague in the WCAG guidance. I come down on the side of we show the yellow halo for native focusable elements only (buttons, links, form inputs, etc.)

## Consider

* Adding an explainer to the accordions to let users know how what 
* Refactoring the HTML of the estimated benefits panel. There's an opportunity for improvement here, using description lists and smart headings to build semantic relationships. The visual could remain as you have proposed, but it would be more meaningful to screen reader users.
* ASK: Do you envision the estimated benefits footer container being sticky at mobile size? If yes, we'll want to test this carefully for mobile + screen reader usability.

**Comment by amyknox** [comment]
@1Copenut - thank you for the input! A number of your recommendations we'll be forwarding to our dev & QA colleagues.  In the short term, we're definitely including  "Use the fields below to calculate your benefits" in the Estimate Your Benefits panel and thinking about how to do usability testing with assistive tech and on mobile.

**Comment by dneel-bah** [comment]
Must and should items are captured in issues: #7842 & #8002

**Comment by dneel-bah** [comment]
@1Copenut The must & should items have been addressed.  It is worth noting, we are going to be tweaking the text for the #1 should item.  Can this issue be closed out?

**Comment by 1Copenut** [comment]
@dneel-bah I verified the must and should items in staging with IE11 + JAWS 2019. They worked as expected for colleges and VETTEC schools. Please feel free to close this at your convenience.
